src/ports/postgres/modules/regress/linear.sql_in

```diff
@@ -183,16 +183,15 @@ FROM (
 @par Prediction Function
 The prediction function is as follows:
 <pre class="syntax">
-linregr_predict(coef, col_ind)
+linregr_predict(col_ind, coef)
```

Comment:
This is inconsistent with the actual function, which has `coef` as the first parameter. I believe the examples incorrectly place the `col_ind` first.

Reply:
Yes, you are correct.  From
http://madlib.apache.org/docs/latest/linear_8sql__in.html

float8 linregr_predict | ( float8 [] | coef, float8 [] | col_ind_var)

I will revert and correct the examples